gridcapa-core-cc-post-processing-app/src/main/java/com/farao_community/farao/core_cc_post_processing/app/services/ZipAndUploadService.java

```diff
@@ -57,7 +57,7 @@ public void zipAndUploadLogs(final List<byte[]> logList,
             for (final byte[] bytes : logList) {
                 ZipUtil.collectAndZip(zos, bytes);
             }
-            zos.close();
+            zos.close(); // NOSONAR because the `zos` ZipOutputStream needs to closed before calling `toByteArray()` method on `baos`
```

Comment:
fix the comment replace ".... needs to closed before ..."  by "..must be closed before ..."